### PR TITLE
Update melody playback

### PR DESCRIPTION
**Pull request by mathiasvr**

### CODE REVIEW

File: src/Components/MelodyEditor/MelodyElement/index.jsx [hunk 1]
```diff
@@ -34,15 +34,18 @@ const MelodyElement = forwardRef(({
...
+  const oscillator = useRef(null);
   const highlighted = useRef(null);
 
   useImperativeHandle(ref, () => ({
     play() {
       playMelody();
     },
     stop() {
-      stop.current = true;
+      if (oscillator.current) {
```
Comment (mathiasvr) [inline src/Components/MelodyEditor/MelodyElement/index.jsx:45]: I suppose I should have just called stopMelody() here...

Comment (stylesuxx) [reply src/Components/MelodyEditor/MelodyElement/index.jsx:45]: Yep.

**Review by stylesuxx — COMMENTED** [review]
@mathiasvr could you just update the stop please? Rest is looking good - thank you. After that I am ready for a lengthy re-base :-D

**Comment by mathiasvr** [comment]
No problem, it should be done now!

**Review by stylesuxx — APPROVED** [review]

**Pull request merged**